locks.py

```diff
@@ -19,7 +19,6 @@
 amd_bbw2_lock = util.MasterLock("amd_bbw2_lock", maxCount=6)
 intel_bbw1_lock = util.MasterLock("intel_bbw1_lock", maxCount=5)
 p9_rhel8_bbw1_lock = util.MasterLock("p9_rhel8_bbw1_lock", maxCount=6)
-p9_rhel7_bbw1_lock = util.MasterLock("p9_rhel7_bbw1_lock", maxCount=6)
```

Comment:
@RazvanLiviuVarzaru here it's an unfortunate naming, there was a power machine that was running rhel7 as the host and used as a worker. This can be removed, but it's not necessary related to the removal of RHEL 7. There was the hope that this machine (together with another one running rhel8 as host - the above one) will come back at some point, but apparently not, so also the line above with rhel8 can be removed.